src/open_inwoner/accounts/models.py

```diff
@@ -313,6 +314,13 @@ def has_verified_email(self):
             "Indicates if the user wants to receive notifications for updates concerning plans and actions."
         ),
     )
+    prev_login = models.DateTimeField(
```

Comment:
Nothing wrong with verbose names!

```suggestion
    previous_login = models.DateTimeField(
```